.github/workflows/ci.yml

```diff
@@ -5,8 +5,20 @@ on:
     branches:
       - develop
   pull_request:
+    branches:
+      - develop
```

Comment:
You can remove this and it should run on pull requests to any branch.

Slightly more canonically, you can change it like this:

```yaml
on:
  - push:
      branches:
        - develop
  - pull_request
```

Or even better might be to get rid of the branch in `push` too:

```yaml
on:
  - push
  - pull_request
```